src/sentry/static/sentry/app/views/settings/components/settingsLayout.jsx

```diff
@@ -17,32 +16,61 @@ class SettingsLayout extends React.Component {
     routes: PropTypes.array,
   };
 
+  state = {
+    navVisible: false,
+  };
+
+  componentWillMount() {
+    this.unlisten = this.props.router.listen(() => {
+      this.toggleNav(false);
+    });
+  }
+  componentWillUnmount() {
+    this.unlisten();
+  }
+
+  toggleNav(navVisible) {
+    this.setState({navVisible});
+    if (navVisible) {
+      document.getElementsByTagName('body')[0].classList.add('modal-open');
```

Comment:
Hmm is this a bootstrap class?

Reply:
Yeah I think it is. Would be open to a better solution here for sure.